src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JmsToKafkaHeaderConverter.java

```diff
@@ -0,0 +1,51 @@
+package com.ibm.eventstreams.connect.mqsource.processor;
+
```

Comment:
This source file is building with compiler warnings because of unchecked or unsafe operations. Please fix.

Reply:
`[WARNING] /Users/andrew/kafka-connect-mq-source/src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JmsToKafkaHeaderConverter.java:[49,51] unchecked method invocation: method list in class java.util.Collections is applied to given types
  required: java.util.Enumeration<T>
  found: java.util.Enumeration
[WARNING] /Users/andrew/kafka-connect-mq-source/src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JmsToKafkaHeaderConverter.java:[49,76] unchecked conversion
  required: java.util.Enumeration<T>
  found:    java.util.Enumeration
[WARNING] /Users/andrew/kafka-connect-mq-source/src/main/java/com/ibm/eventstreams/connect/mqsource/processor/JmsToKafkaHeaderConverter.java:[50,40] unchecked call to forEach(java.util.function.Consumer<? super E>) as a member of the raw type java.util.ArrayList`